vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/di/Modules.kt

```diff
@@ -13,14 +12,15 @@ import com.malinskiy.marathon.cache.test.key.ComponentCacheKeyProvider
 import com.malinskiy.marathon.execution.ComponentInfoExtractor
 import com.malinskiy.marathon.execution.TestParser
 import com.malinskiy.marathon.io.CachedFileHasher
+import com.malinskiy.marathon.io.Md5FileHasher
 import com.malinskiy.marathon.log.MarathonLogConfigurator
 import com.malinskiy.marathon.report.logs.LogsProvider
 import org.koin.dsl.module
 
 val androidModule = module {
     single<TestParser?> { AndroidTestParser() }
     single<ComponentInfoExtractor?> { AndroidComponentInfoExtractor() }
-    single<ComponentCacheKeyProvider?> { AndroidComponentCacheKeyProvider(CachedFileHasher(ApkFileHasher())) }
+    single<ComponentCacheKeyProvider?> { AndroidComponentCacheKeyProvider(CachedFileHasher(Md5FileHasher())) }
```

Comment:
We use MD5 in `AndroidAppInstaller.isApkInstalled` and in `AndroidAppInstaller.getHashOnDevice`, so ApkFileHasher is doing nothing. Checking MD5 is really fast and I don't see that it takes even more than a second on the build. We also have a hashmap so it is calculated only once.